crates/math/src/bytable.rs

```diff
@@ -179,5 +223,74 @@ mod tests {
             let recovered = Uint256::from_le_bytes(number.to_le_bytes());
             prop_assert_eq!(number, recovered);
         }
+
+        /// Ensure the bytable methods work for `Int128`.
+        #[test]
+        fn integer_bytable_works_i128(number in any::<i128>()) {
+            let number = Int128::from(number);
+
+            // Convert the number to big endian bytes and back, should get the
+            // the same value
+            let recovered = Int128::from_be_bytes(number.to_be_bytes());
+            prop_assert_eq!(number, recovered);
+
+            // Same thing for little endian
+            let recovered = Int128::from_le_bytes(number.to_le_bytes());
+            prop_assert_eq!(number, recovered);
+        }
+
+        /// Ensure the bytable methods work for `Int256`.
+        #[test]
+        fn integer_bytable_works_i256(number in any::<i128>()) {
+            let number = Int256::from(number);
```

Comment:
This test only tests integers within the i128 range. I'm going to modify it to test the full i256 range using the same approach as in `integer_bytable_works_u256`.